code/modules/mob/living/say.dm

```diff
@@ -18,7 +18,7 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 
 	// Faction
 	"t" = "Syndicate",
-	"y" = "CentCom",
+	"y" = "Horizons",

```

Comment:
This should be `Head Office`, or whatever you set the define to before. I think.

Reply:
Looks like you fixed.